### PR TITLE
How will Edge Intelligence Benefit IoT?

### LINKED ISSUE #3672: How will Edge Intelligence Benefit IoT?
### Introduction paragraph (2-3 paragraphs):
With Edge AI, IoT devices are becoming smarter. With machine learning, edge devices are now able to make decisions. They can make predictions, process complex data, and administer solutions.

For example, edge IoT devices can process operating conditions to predict if a given piece of machinery will fail. This allows companies to perform predictive maintenance and avoid the larger damages and costs that would have been incurred in the event of a complete failure.

Also, a security camera equipped with edge AI may no longer only capture video. We will be able to identify humans and count foot traffic. Or, with facial recognition, even identify exactly who has passed through an area and when.
### Key takeaways: 

1. Why edge brings on the table (reduced bandwidth requirement, reduced latency, increased reliability)
2. Chip-enabled intelligent edge
3. Edge intelligence and smart cameras on the IoT

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.
N/A
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by mumbi00**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [ ] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [ ] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #3672

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by LinusMuema** [comment]
plagiarism check done :heavy_check_mark:

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+Both the long-established chips makers and startups are focusing on adding artificial intelligence capabilities to the edge. The startups include [Mythic](https://www.mythic-ai.com/), [Hailo](https://hailo.ai/), [Graphcore](https://www.graphcore.ai/), [EdgeQ](https://edgeq.io/), [Flex Logic](https://flex-logix.com/), [Deep Vision](https://deepvision.io/), [BrainChip](https://brainchipinc.com/), and [Blaize](https://www.blaize.com/). [STMelectronics](https://www.st.com/content/st_com/en.html), [NXP](https://www.nxp.com/), [NVIDIA](https://www.nvidia.com/en-us/), [Maxim](https://www.maxim.com/), [Intel](https://www.intel.com/content/www/us/en/homepage.html), [ARM](https://www.arm.com/), [AMD](https://www.amd.com/en), and [Qualcomm](https://www.qualcomm.com/) have been making microcontrollers and microprocessors in the edge AI for quite some time now.
+
+#### Benefits of chip-enabled intelligent edge
+Chip-enabled edge intelligence improves the value of IoT devices in many ways. Below are some of the benefits:
+- Edge AI chips generate less heat and use less power. They are integrable with handheld devices like smartphones and other non-consumer devices like robots.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:64]:
```suggestion
Edge AI chips generate less heat and use less power. They can be integrated with handheld devices. Like, smartphones and other non-consumer devices like robots.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- Edge-based AI chips reduce or eliminate the need to transmit bulk data to a cloud solution or data center. This means processor-intensive machine learning computations happen locally, ensuring data security, privacy, and high processing speed.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:65]:
```suggestion
Edge-based AI chips reduce or end the need to send bulk data to a cloud solution or data center. This means processor-intensive machine learning computations happen locally. This promotes data security, privacy, and high processing speed.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+- Edge intelligence in 5G networks allows safe transportation or storage of critical data in a central location.
+- Edge AI chips help companies to collect data from connected devices and analyze the data at the device. Thus, avoiding the complexity, security challenges, and costs associated with sending and analyzing data on the cloud.
+
+### Edge intelligence and smart cameras on the IoT
+Smart cameras with artificial intelligence, IP connectivity, and advanced data analytics drive IoT innovations and new use cases. Chips on AI-enabled smart cameras empower IoT devices designers to create new solutions and products based on the power of edge artificial intelligence.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:70]:
```suggestion
Smart cameras with AI, IP connectivity, advanced, and data analytics drive IoT innovations and new use cases. Chips on AI-enabled smart cameras empower IoT devices designers to create new solutions. And, products based on the power of edge artificial intelligence.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- Edge-based AI chips reduce or eliminate the need to transmit bulk data to a cloud solution or data center. This means processor-intensive machine learning computations happen locally, ensuring data security, privacy, and high processing speed.
+- Edge intelligence in 5G networks allows safe transportation or storage of critical data in a central location.
+- Edge AI chips help companies to collect data from connected devices and analyze the data at the device. Thus, avoiding the complexity, security challenges, and costs associated with sending and analyzing data on the cloud.
+
+### Edge intelligence and smart cameras on the IoT
+Smart cameras with artificial intelligence, IP connectivity, and advanced data analytics drive IoT innovations and new use cases. Chips on AI-enabled smart cameras empower IoT devices designers to create new solutions and products based on the power of edge artificial intelligence. 
+
+Video conferencing devices, dashboard cameras, and surveillance cameras now have new use cases. Such as face recognition, people counting, body detection, license plate recognition, and object segmentation and classification.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:72]:
```suggestion
Video conferencing devices, dashboard cameras, and surveillance cameras now have new use cases. Such as facial recognition, people counting, body detection, license plate recognition. Also, object segmentation and classification.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+Dual-facing edge AI dash cameras help in driver behavior analysis, fleet management, and street condition analysis. They record drivers' anomalous behaviors such as texting or talking on the phone, dozing off, drinking, and eating and send notifications and alerts of such recurring instances. All these are key to generating drivers' scorecards and lowering the risks of accidents.
+
+In fleet management, these cameras record footage of unexpected braking, collisions, sudden accelerations, and so on. Such footage is uploaded to cloud storage automatically for later viewing.
+
+Waste management companies have traditionally maintained mounted cameras on garbage trucks. But they are now replacing these cameras with edge AI cameras that can identify items such as metal scrap, plastic, Styrofoam, cardboard, and paper. These cameras can also identify inappropriate and hazardous waste. This has helped improve waste disposal productivity and classification.
+
+### Conclusion
+Edge intelligence is transforming the internet of things industry in a big way. It has made real-time data analysis easier, helped increase operational efficiency, and reduced data handling and storage-related costs.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:88]:
```suggestion
Edge intelligence is transforming the internet of things industry in a big way. It has made real-time data analysis easier. Also, it helped increase operational efficiency, and reduced data handling and storage-related costs.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Edge AI cameras
+AI dashboard-mounted cameras use edge processing for the detection of objects and events in real-time. This has helped improve the security, visibility, and safety of drivers, passengers, and pedestrians and reduce vehicle accidents.
+
+Edge AI cameras are built to run image-based deep learning and machine learning models at the local device. The dual-facing edge AI dash camera is common in parking lot management and traffic monitoring. Other edge AI cameras are used in sports broadcasting, smart farming, garbage management, and crowd analysis and monitoring.
+
+Dual-facing edge AI dash cameras help in driver behavior analysis, fleet management, and street condition analysis. They record drivers' anomalous behaviors such as texting or talking on the phone, dozing off, drinking, and eating and send notifications and alerts of such recurring instances. All these are key to generating drivers' scorecards and lowering the risks of accidents.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:81]:
```suggestion
Dual-facing edge AI dash cameras help in driver behavior analysis, fleet management, and street condition analysis. They record drivers' anomalous behaviors. Such as texting or talking on the phone, dozing off, drinking, and eating and send notifications and alerts of such recurring instances. All these are key to generating drivers' scorecards and lowering the risks of accidents.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+In fleet management, these cameras record footage of unexpected braking, collisions, sudden accelerations, and so on. Such footage is uploaded to cloud storage automatically for later viewing.
+
+Waste management companies have traditionally maintained mounted cameras on garbage trucks. But they are now replacing these cameras with edge AI cameras that can identify items such as metal scrap, plastic, Styrofoam, cardboard, and paper. These cameras can also identify inappropriate and hazardous waste. This has helped improve waste disposal productivity and classification.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:85]:
```suggestion
Waste management companies have traditionally maintained mounted cameras on garbage trucks. But they are now replacing these cameras with edge AI cameras. They can identify items such as metal scrap, plastic, Styrofoam, cardboard, and paper. These cameras can also identify inappropriate and hazardous waste. This has helped improve waste disposal productivity and classification.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-will-edge-intelligence-benefit-iot/hero.jpg
+    alt: How will Edge Intelligence Benefit IoT? Hero Image 
+---
+With [edge artificial intelligence (AI)](/engineering-education/edge-artificial-intelligence-and-mobile-development/) internet of things (IoT) devices are becoming smarter. Edge devices now have machine learning capabilities that empower them to process complex data locally, administer solutions and make predictions. 
+<!--more-->
+For instance, if you install edge IoT devices in your production plant, you can monitor the operating conditions of the machinery. With this, you'll perform predictive maintenance and avoid complete system failure, associated damages, and costs.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:19]:
```suggestion
For instance, if you install edge IoT devices in your production plant, you can track the operating conditions of the machinery. With this, you will perform predictive maintenance, avoid complete system failure, associated damages, and costs.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+Smart cameras equipped with edge AI chips have uncovered new use cases beyond capturing video. Such cameras can identify human traffic, monitor divers' behavior, count foot traffic, classify and segment objects, etc.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:21]:
```suggestion
Smart cameras equipped with edge AI chips have uncovered new use cases beyond capturing video. Such cameras can identify human traffic, track divers' behavior and count foot traffic. They can also classify and segment objects.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Benefits of chip-enabled intelligent edge
+Chip-enabled edge intelligence improves the value of IoT devices in many ways. Below are some of the benefits:
+- Edge AI chips generate less heat and use less power. They are integrable with handheld devices like smartphones and other non-consumer devices like robots.
+- Edge-based AI chips reduce or eliminate the need to transmit bulk data to a cloud solution or data center. This means processor-intensive machine learning computations happen locally, ensuring data security, privacy, and high processing speed.
+- Edge intelligence in 5G networks allows safe transportation or storage of critical data in a central location.
+- Edge AI chips help companies to collect data from connected devices and analyze the data at the device. Thus, avoiding the complexity, security challenges, and costs associated with sending and analyzing data on the cloud.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:67]:
```suggestion
Edge AI chips help companies to collect data from connected devices and analyze the data at the device. Thus, avoiding the complexity and security challenges. And, costs associated with sending and analyzing data on the cloud.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+### Edge intelligence and smart cameras on the IoT
+Smart cameras with artificial intelligence, IP connectivity, and advanced data analytics drive IoT innovations and new use cases. Chips on AI-enabled smart cameras empower IoT devices designers to create new solutions and products based on the power of edge artificial intelligence. 
+
+Video conferencing devices, dashboard cameras, and surveillance cameras now have new use cases. Such as face recognition, people counting, body detection, license plate recognition, and object segmentation and classification.
+
+Chips on these cameras ensure that workloads like image processing, security, machine learning, and computer vision are kept at the edge device. Manufacturers of IoT devices have found a solution to network latency and user privacy and are now focusing resources on innovation.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:74]:
```suggestion
Chips on these cameras ensure that workloads like image processing, security, machine learning, and computer vision are kept at the edge device. Manufacturers of IoT devices have found a solution to network latency and user privacy.  And, are now focusing resources on innovation.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Edge AI cameras
+AI dashboard-mounted cameras use edge processing for the detection of objects and events in real-time. This has helped improve the security, visibility, and safety of drivers, passengers, and pedestrians and reduce vehicle accidents.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:77]:
```suggestion
AI dashboard-mounted cameras use edge processing for the detection of objects and events in real-time. These have helped improve the security, visibility, and safety of drivers, passengers, and pedestrians. Hence, reducing vehicle accidents.
```

File: content/articles/how-will-edge-intelligence-benefit-iot/index.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+Edge AI cameras are built to run image-based deep learning and machine learning models at the local device. The dual-facing edge AI dash camera is common in parking lot management and traffic monitoring. Other edge AI cameras are used in sports broadcasting, smart farming, garbage management, and crowd analysis and monitoring.
```
Comment (Dawe-7) [inline content/articles/how-will-edge-intelligence-benefit-iot/index.md:79]:
```suggestion
Edge AI cameras are built to run image-based deep learning and machine learning models at the local device. The dual-facing edge AI dash camera is common in parking lot management and traffic monitoring. Other edge AI cameras are used in sports broadcasting, smart farming and garbage management. They are also used in crowd analysis and monitoring.
```

**Comment by Dawe-7** [comment]
> @Dawe-7 Why close the PR? You assigned this article 14 days ago, delayed and only reviewed it 3 days ago. I think you've not given enough time to revise. CC @hectorkambow

@mumbi00, We only accept polished up articles and close if the article needs revisions by the author. Kindly work on this PR and ping me when you are done. I'll reopen the PR once changes have been made.

cc: @hectorkambow

**Comment by mumbi00** [comment]
@Dawe-7  Can you re-open this PR so that I can be able to commit changes.

**Comment by mumbi00** [comment]
@Dawe-7  Revision is complete.

**Comment by hectorkambow** [comment]
Overall nice work 👍

**Pull request closed without merging**